afs/afs-core/src/main/java/com/powsybl/afs/AppFileSystem.java

```diff
@@ -43,7 +45,7 @@
 
     private final ListenableAppStorage storage;
 
-    private final NodeInfo rootNodeInfo;
+    private final Supplier<NodeInfo> rootNodeInfo;
```

Comment:
I propose not to fix it as it would require  to update guava dependency which is always tricky (because all librairies depends on it and there is no backward compat). In latest guava version guava supplier inherit of java 8 supplier, si this warning can be avoided. We will replace all guava supplier once guava update will be done later on.